internal/service/domain/tree/tree.go

```diff
@@ -121,32 +124,43 @@ func (s *TreeService) Create(ctx context.Context, treeCreate *entities.TreeCreat
 		return nil, service.NewError(service.BadRequest, errors.Wrap(err, "validation error").Error())
 	}
 
-	fn := make([]entities.EntityFunc[entities.Tree], 0)
+	var treeClusterID *entities.TreeCluster
```

Comment:
I think the name `treeClusterID` can be misleading as it holds an actual tree cluster entity. I don't want to be the guy who complains about variable names, but the name is not well chosen